SAP/jenkins-library
# Improve neo deploy

# Changes


Details:
- Locking
- New parameters: environment, vmArguments
- Assert password does not start with @
- Link to cloud cockpit
- Only execute rolling update if app is running
- Show logs if deployment failed
- Restart app after normal deployment
- Use neo namespace for parameters
- Align parameter names with neo sdk: size, application, source
- Remove vmSize check

fixes #429
